pom.xml

```diff
@@ -143,7 +143,7 @@
       <!-- bundle: com.google.cloud.tools.appengine -->
       <groupId>com.google.cloud.tools</groupId>
       <artifactId>appengine-plugins-core</artifactId>
-      <version>0.3.5</version>
+      <version>0.3.6-SNAPSHOT</version>
```

Comment:
No SNAPSHOT. BTW, 0.3.6 is live: http://repo1.maven.org/maven2/com/google/cloud/tools/appengine-plugins-core/0.3.6/